Dockerfile

```diff
@@ -0,0 +1,29 @@
+# syntax=docker/dockerfile:1.3-labs
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.6
+
+COPY <<EOF /etc/yum.repos.d/mongodb-org-x86_64-6.0.repo
+[mongodb-org-x86_64-6.0]
+name=MongoDB Repository
+baseurl=https://repo.mongodb.org/yum/redhat/8/mongodb-org/6.0/x86_64/
+gpgcheck=1
+enabled=1
+gpgkey=https://www.mongodb.org/static/pgp/server-6.0.asc
+EOF
+
+COPY <<EOF /etc/yum.repos.d/mongodb-org-aarch64-6.0.repo
+[mongodb-org-aarch64-6.0]
+name=MongoDB Repository
+baseurl=https://repo.mongodb.org/yum/redhat/8/mongodb-org/6.0/aarch64/
+gpgcheck=1
+enabled=1
+gpgkey=https://www.mongodb.org/static/pgp/server-6.0.asc
+EOF
```

Comment:
I see that the current dockerfile uses the cli version from the official mongoDB repository. Do we want to provide another dockerfile to download the cli repo and build the binary with the makefile? In this way, we will be able to test the unreleased cli versions as well.

tagging @matt-condon as I don't think this was agreed upon in the scope. (As a result, the answer "It is not in the scope so we won't do it" is more than acceptable here.)

Reply:
> Do we want to provide another dockerfile to download the cli repo and build the binary with the makefile?

For local dev experience or for validation when the Dockerfile changes?

I think this could be considered a nice-to-have
I'll create a low-priority story for this and backlog it later if we consider it out-of-scope

Reply:
> For local dev experience or for validation when the Dockerfile changes?

for both

> I think this could be considered a nice-to-have
> I'll create a low-priority story for this and backlog it later if we consider it out-of-scope

Sure. Thanks!

Reply:
https://jira.mongodb.org/browse/CLOUDP-188091

Reply:
Something to keep in mind when using the CLI from the MongoDB repo (this is related to [CLOUDP-187063: Override agent for Dockerized AtlasCLI](https://jira.mongodb.org/browse/CLOUDP-187063)) : we inject the information shown in the `UserAgent` field  at compilation time during the release [.atlascli.goreleaser.yml](https://github.com/mongodb/mongodb-atlas-cli/blob/09a1900f0c02caa2df2a8321bec4eb413393786a/build/package/.atlascli.goreleaser.yml#L17) or when running [make build-atlascli](https://github.com/mongodb/mongodb-atlas-cli/blob/09a1900f0c02caa2df2a8321bec4eb413393786a/Makefile#L135). If you use the CLI from the MongoDB repo, you won't be able to override these variables when running in Docker.

Reply:
One workaround may be to set the default value of [ToolName](https://github.com/mongodb/mongodb-atlas-cli/blob/5d4dd4bf038f41d7496e353f4a6083c0aa74e51b/internal/config/profile.go#L75C2-L75C10) to `atlascli-container` instead of `MongoCLI` but this should be investigated.

Reply:
Yes thanks for the pointers, I will investigate that as part of https://jira.mongodb.org/browse/CLOUDP-187063